src/tokens/ERC20Gauges.sol

```diff
@@ -104,78 +89,14 @@ abstract contract ERC20Gauges is ERC20 {
     EnumerableSet.AddressSet internal _deprecatedGauges;
 
     /*///////////////////////////////////////////////////////////////
-                              VIEW HELPERS
+                            VIEW HELPERS
     //////////////////////////////////////////////////////////////*/
 
-    /// @notice return the end of the current cycle. This is the next unix timestamp which evenly divides `gaugeCycleLength`
-    function getGaugeCycleEnd() public view returns (uint32) {
-        return _getGaugeCycleEnd();
-    }
-
-    /// @notice see `getGaugeCycleEnd()`
-    function _getGaugeCycleEnd() internal view returns (uint32) {
-        uint32 nowPlusOneCycle = block.timestamp.safeCastTo32() +
-            gaugeCycleLength;
-        unchecked {
-            return (nowPlusOneCycle / gaugeCycleLength) * gaugeCycleLength; // cannot divide by zero and always <= nowPlusOneCycle so no overflow
-        }
-    }
-
-    /// @notice returns the current weight of a given gauge
-    function getGaugeWeight(address gauge) public view returns (uint112) {
-        return _getGaugeWeight[gauge].currentWeight;
-    }
-
-    /// @notice returns the stored weight of a given gauge. This is the snapshotted weight as-of the end of the last cycle.
-    function getStoredGaugeWeight(address gauge) public view returns (uint112) {
-        if (_deprecatedGauges.contains(gauge)) return 0;
-        return _getStoredWeight(_getGaugeWeight[gauge], _getGaugeCycleEnd());
-    }
-
-    /// @notice see `getStoredGaugeWeight()`
-    function _getStoredWeight(
-        Weight storage gaugeWeight,
-        uint32 currentCycle
-    ) internal view returns (uint112) {
-        return
-            gaugeWeight.currentCycle < currentCycle
-                ? gaugeWeight.currentWeight
-                : gaugeWeight.storedWeight;
-    }
-
-    /// @notice returns the current total allocated weight
-    function totalWeight() external view returns (uint112) {
-        return _totalWeight.currentWeight;
-    }
-
-    /// @notice returns the stored total allocated weight
-    function storedTotalWeight() external view returns (uint112) {
-        return _getStoredWeight(_totalWeight, _getGaugeCycleEnd());
-    }
-
     /// @notice returns the set of live gauges
     function gauges() external view returns (address[] memory) {
         return _gauges.values();
     }
 
-    /** 
-      @notice returns a paginated subset of live gauges
-      @param offset the index of the first gauge element to read
-      @param num the number of gauges to return
-    */
-    function gauges(
-        uint256 offset,
-        uint256 num
-    ) external view returns (address[] memory values) {
-        values = new address[](num);
-        for (uint256 i = 0; i < num; ) {
-            unchecked {
-                values[i] = _gauges.at(offset + i); // will revert if out of bounds
-                ++i;
-            }
-        }
-    }
-
```

Comment:
I think this is helpful for the frontend to bound the numbers of gauges returned.

Reply:
We started working on the front-end and API back-end, and concluded that this function isn't useful, so I wanted to clear out the dead code (same reason we clear cycle-based logic). What takes time is doing an RPC call, not the quantity of data returned by the RPC call. And the quantity of gauges we expect to have is definitely <1000 so the quantity of data returned by a normal `.gauges()` call is still pretty small.